validation/tsconfig.json

```diff
@@ -19,13 +19,11 @@
     "esModuleInterop": true
   },
   "include": [
-    "compare.ts"
+    "src/**/**/*"
```

Comment:
From https://www.typescriptlang.org/tsconfig#include, `**/ matches any directory nested to any level`. So `**/**/` I guess matches any directory nested to >2 levels. Doesn't this exclude things like `src/server.ts`?

Reply:
Interestingly it does seem to include `server.ts`, maybe because it's in `src/`? By the looks of it though I could replace `src/**/**/*` with `src/**/*` and it would have the same effect?